generative/2d_ldm/README.md

```diff
@@ -0,0 +1,139 @@
+# 2D Latent Diffusion Example
+This folder contains an example for training and validating a 2D Latent Diffusion Model on Brats axial slices. The example includes support for multi-GPU training with distributed data parallelism.
+
+The workflow of the Latent Diffusion Model is depicted in the figure below. It begins by training an autoencoder in pixel space to encode images into latent features. Following that, it trains a diffusion model in the latent space to denoise the noisy latent features. During inference, it first generates latent features from random noise by applying multiple denoising steps using the trained diffusion model. Finally, it decodes the denoised latent features into images using the trained autoencoder.
+<p align="center">
+  <img src="./figs/ldm.png" alt="latent diffusion scheme")
+</p>
+
+MONAI latent diffusion model implementation is based on the following papers:
+
+[**Latent Diffusion:** Rombach, Robin, et al. "High-resolution image synthesis with latent diffusion models." CVPR 2022.](https://openaccess.thecvf.com/content/CVPR2022/papers/Rombach_High-Resolution_Image_Synthesis_With_Latent_Diffusion_Models_CVPR_2022_paper.pdf)
+
+This network is designed as a demonstration to showcase the training process for this type of network using MONAI. To achieve optimal performance, it is recommended that users have a GPU with memory larger than 32G to accommodate larger networks and attention layers.
+
+### 1. Data
+
+The dataset we are experimenting with in this example is BraTS 2016 and 2017 data.
+
+BraTS is a public dataset of brain MR images. Using these images, the goal is to generate images that look similar to the images in BraTS 2016 and 2017 dataset.
+
+The data can be automatically downloaded from [Medical Decathlon](http://medicaldecathlon.com/) at the beginning of training.
+
+Disclaimer: We are not the host of the data. Please make sure to read the requirements and usage policies of the data and give credit to the authors of the dataset! We acknowledge the National Cancer Institute and the Foundation for the National Institutes of Health, and their critical role in the creation of the free publicly available LIDC/IDRI Database used in this study.
+
+### 2. Installation
+```
+pip install lpips
+pip install git+https://github.com/Project-MONAI/GenerativeModels.git#egg=Generative
```

Comment:
Hi @Can-Zhao @wyli @Nic-Ma , is it possible to request a dedicated branch in the GenerativeModels repo?

Otherwise, I would be worried the tutorials would depend on no breaking changes or bug in the dev/main branch.

We can possibly follow metrics_reloaded and make a monai-support branch?
https://github.com/Project-MONAI/tutorials/blob/main/modules/metrics_reloaded/README.md

Reply:
cc @ericspod

Reply:
We've created a tag for the current version of the repo you can refer to in the pip line: `pip install git+https://github.com/Project-MONAI/GenerativeModels.git@0.2.0` We can create tags at any time if you need to mark a specific version of the repo so just ask when that's the case.

Reply:
Thank you!